setup-services.sh

```diff
@@ -116,14 +117,15 @@ fi
       sed -e '/gemspec/a gem "ruby-dbus", path: "/checkout-ruby-dbus"' -i Gemfile
   fi
 
-  if [ -n "$CI" ]; then
+  if [ -n "${CI:-}" ]; then
     # in CI reuse the pre-installed system gems from RPMs
     bundle config set --local disable_shared_gems 0
+    sudo bundle install
```

Comment:
it should use `$SUDO` see https://github.com/agama-project/agama/pull/1687/files#diff-327a0d405c7c05ecb8395935997d54a646315e7da5e275748c565467fc3b7f61R17